test/e2e/framework/pod/wait.go

```diff
@@ -226,13 +231,20 @@ func WaitForPodCondition(c clientset.Interface, ns, podName, desc string, timeou
 	return fmt.Errorf("Gave up after waiting %v for pod %q to be %q", timeout, podName, desc)
 }
 
+func podTimeout() time.Duration {
+	if framework.NodeOSDistroIs("windows") {
```

Comment:
Does this mean we don't run e2e tests on mixed linux + windows node clusters?

Reply:
In general we do not.

Reply:
it is a mixed cluster, but in test config for Windows testing, we pass Windows as nodeOSDistro.